x-pack/plugins/canvas/public/functions/timelion.js

```diff
@@ -88,14 +71,10 @@ export const timelion = () => ({
       },
     };
 
-    return fetch(`${handlers.serverUri}/api/timelion/run`, {
+    return fetch(chrome.addBasePath(`/api/timelion/run`), {
       method: 'POST',
       responseType: 'json',
-      headers: {
-        ...handlers.httpHeaders,
```

Comment:
Note: browser functions get a different set of handlers, and `httpHeaders` isn't one of them. There's also no reason to keep this around now that it's a browser fetch call.

Reply:
Can you remove `httpHeaders` from the server function handlers?  https://github.com/elastic/kibana/pull/27944/files#diff-90559938fbd2ac2a52a86d6eb5fefe90R34

I'm pretty certain this was only used for `timelion`.

Reply:
Same goes for `serverUri` in the server handlers object.

Reply:
I think `serverUri` is still useful since you need to know that to hit Kibana APIs and we want to allow plugins flexibility without needing to write actual Kibana plugins.

Reply:
I'm going to make this change in a new PR actually.

Reply:
Opened https://github.com/elastic/kibana/pull/28011